src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
```

Comment:
https://dev.to/clairecodes/my-misconceptions-about-the-universal-selector-in-css-4ngm